### DISCUSSION TITLE
How do I download this file？

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#885] [Questions & Problems] [answered]
![image](https://user-images.githubusercontent.com/81560924/113838317-3d6d7d00-97c1-11eb-8b07-4b91b0bef39c.png)

**Comment by PavelBal** [comment, marked as answer]
Right-click on the file in the folder "Ex_1 and 2_ADC..." and select "Save as..." (the name of the entry may depend on the browser you are using).

**Comment by Achineseman** [comment]
> 右键单击文件夹中的文件"Ex_1和2_ADC..."并选择"保存为。。。"（条目的名称可能取决于您正在使用的浏览器）。

Thank you for helping me
